sdk/monitor/Azure.Monitor.Query/src/LogsQueryClientBuilderExtensions.cs

```diff
@@ -12,7 +12,7 @@ public static class LogsQueryClientBuilderExtensions
     {
         /// <summary> Registers a <see cref="LogsQueryClient"/> instance. </summary>
         /// <param name="builder"> The builder to register with. </param>
-        /// <param name="endpoint"> The resource manager service endpoint to use. For example <c>https://management.azure.com/</c> for public cloud. </param>
+        /// <param name="endpoint"> The service endpoint to use. Default is 'https://api.loganalytics.io'. </param>
```

Comment:
This isn't really the default in this context because this is a required parameter, this raises the question - should this be optional?

Reply:
I think the endpoint can vary based on what cloud you are using. For example I think government clouds have a different endpoint

Reply:
But we expose a constructor in the client that doesn't take an endpoint, so should we do the same here?